### DISCUSSION TITLE
Can't install dev build

### DISCUSSION THREAD

**Discussion by Iaotle** [File-New-Project/EarTrumpet#1237] [Q&A]
Couldn't find a way to fix this. 

![image](https://user-images.githubusercontent.com/43072246/193434756-047ef658-60f0-4578-8765-9c7a91ded5af.png)

**Comment by riverar** [comment]
@Iaotle Sounds like some Microsoft root certificate authorities are missing from your machine. Can you download the appxbundle manually (https://install.eartrumpet.app/dev/EarTrumpet.Package_2.2.0.6_x86.appxbundle), right-click the file, then go to Properties > Digital Signatures > View Signature > Certification Path and take/provide a screenshot?

**Comment by Iaotle** [comment]
Sure:
![image](https://user-images.githubusercontent.com/43072246/193485144-af822972-48e9-4efe-80fe-576c3e5e16a3.png)


By the way, previously I tried installing it from the website but that installer didn't have any certificates with it.

**Reply by Iaotle** [reply]
PS: I tried installing the ceritficates in the path but it didn't do anything unfortunately.

**Reply by riverar** [reply]
Hm odd. What build of Windows? And can you click the little i in the corner of the App Installer window to get its version? Thanks!

![image](https://user-images.githubusercontent.com/475132/193490071-1efa7ecf-dcb6-405f-9002-5b2f087b25ca.png)

**Reply by Iaotle** [reply]
Sorry for the late reply. I did update it from the MS store and it works now :)
